cob_bms_driver/src/fake_bms.py

```diff
@@ -24,8 +25,15 @@ def __init__(self):
         self.temperature        = 0.0
         self.charging_state     = False
 
-        rospy.Timer(rospy.Duration(0.2), self.timer_cb)
-        rospy.loginfo('FakeBMS is running')
+        try:
```

Comment:
`try-except` should not be necessary in case you use the `default_value` notation